modules/nf-core/ilastik/multicut/main.nf

```diff
@@ -0,0 +1,38 @@
+process ILASTIK_MULTICUT {
+    tag "$meta.id"
+    label 'process_low'
+
+    container "labsyspharm/mcmicro-ilastik:1.6.0"
+
+    input:
+    tuple val(meta), path(h5)
+    path ilp
+    path probs
+
+    output:
+    tuple val(meta), path("*.tiff") , emit: out_tiff
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    /ilastik-release/run_ilastik.sh \\
+        --headless \\
+        --readonly 1 \\
```

Comment:
Are these mandatory arguments? And is there no way this module can be run without these arguments? Otherwise I would suggest removing them here and using `ext.args` to add these.

Reply:
Yes, these are in fact mandatory arguments. `--headless` is required to execute ilastik in command line mode and `--readonly 1` is basically to ensure, that the same model could be used by multiple nextflow processes.

Can these stay as is if mandatory or do I still need to move them somewhere else?

Reply:
They can stay there if they will always be like this :)